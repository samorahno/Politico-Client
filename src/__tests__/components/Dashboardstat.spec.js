import React from 'react';
import { shallow } from 'enzyme';
import DashboardStat from '../../components/common/dashboardstat';

describe('Dashboard Status Component', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<DashboardStat />);
        expect(component).toMatchSnapshot();
    });
});
