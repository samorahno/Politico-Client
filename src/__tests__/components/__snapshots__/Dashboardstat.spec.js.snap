// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Status Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
